src/components/Container/Container.stories.tsx

```diff
@@ -0,0 +1,29 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React from 'react';
+import { Meta, StoryFn } from '@storybook/react';
+import { Container as ContainerComponent, ContainerProps, PaddingType } from './Container';
+
+const acceptablePaddingInput: PaddingType[] = [0, 4, 8, 12, 16, 20, 24, 28, 32, 36, 40];
+
+export default {
+    title: 'Layout/Container',
+    component: ContainerComponent,
+    tags: ['autodocs'],
+    argTypes: {
+        padding: {
+            options: Object.values(acceptablePaddingInput),
+            control: { type: 'select' },
+        },
+        children: { table: { disable: false } },
+    },
+    args: {
+        minWidth: 50,
+        maxWidth: 100,
+        padding: 12,
+        ['data-test-id']: 'custom-test-id',
+        children: <div>This is a container</div>,
```

Comment:
Just an idea, in the example, can you even add a background color to the content so it's easier to see the padding changes.